agent/acs/model/api/api-2.json

```diff
@@ -235,6 +236,20 @@
       "type":"list",
       "member":{"shape":"ElasticNetworkInterface"}
     },
+    "EncodedString":{
```

Comment:
I don't see this used anywhere?

Reply:
yea, it's not part of this feature. we should get rid of it.